### PR TITLE
A situation based problem

### PR DESCRIPTION
**Pull request by anisha282000**
This script is based on the problem i.e. you are given 8 identical looking balls. One of them is heavier than the rest of the 7 (all the others weigh exactly the same). You have to find a different ball.

### CODE REVIEW

**Review by powerexploit — COMMENTED** [review]
Try to remove extra four lines.

**Comment by anisha282000** [comment]
I have removed some extra lines from it.

File: Scripts/8balls.py [hunk 1]
```diff
@@ -1,32 +1,26 @@
...
     elif b2>b1:
      print("b2 is the different ball")
     elif b3>b4:
-      print("b3 is the different ball")
+     print("b3 is the different ball")
     elif b4>b3:
-      print("b4 is the different ball")
+     print("b4 is the different ball")
+    elif group1>group:
+     b5>b6
```
Comment (powerexploit) [inline Scripts/8balls.py]: Solve indentation error do proper indentation.

**Review by powerexploit — COMMENTED** [review]
Solve indentation error do proper indentation.

**Pull request closed without merging**